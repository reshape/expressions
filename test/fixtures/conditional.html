<p>x</p>
<if condition="foo === 'bar'">
  <p>it works!</p>
</if>
<elseif condition="foo === 'wow'">
  <p>it doesn't work</p>
</elseif>
<elseif condition="foo === 'snargle'">
  <p>it really doesn't work</p>
</elseif>
<else>
  <p>it definitely doesn't work</p>
</else>
<p>x</p>
