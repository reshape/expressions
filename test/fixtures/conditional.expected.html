<p>x</p>

  <p>it works!</p>
<p>x</p>
